packages/firebase_ml_vision/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.10.1
+
+* Add `rawBytes` property to `Barcode`.
```

Comment:
The changelog is auto-generated, could you remove this?

Reply:
Done :)